### ISSUE TITLE
Update du type de retour de Json!()

### ISSUE THREAD

**Issue by s18alg** [AAAsso/webservice-emote-mattermost#1]
Le dev de rocket m'a confirmé que le type de retour de json!() va changer dans le future.

Il faudra update l'application le moment venu.

**Comment by DarckCrystale** [comment]
Tu veux dire qu'il faudra la refaire en RUST ? :slightly_smiling_face:

**Comment by s18alg** [comment]
Nope, l'api est déjà un peu en rust, mais une des lib que j'utilise vas subbire un gros update qui vas casser ce que j'ai écris.
